content/outcomes/cloud-native-development/general__using_cloud_foundry.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Using Tanzu Application Service (TAS)
+weight: 50
+layout: single
+team:
+ - VMware Tanzu Labs
+---
+
+VMware [Tanzu Application Service](https://tanzu.vmware.com/application-service) (TAS), formerly Pivotal Cloud Foundry (PCF), is a modern application platform that provides a best-in-class developer experience. It is based on Cloud Foundry, an open-source cloud app platform providing a choice of clouds, developer frameworks, and app services.
+
+While you will be spending most of your time working on software code, it is essential to know how to use the VMware Tanzu Application Service (TAS) platform to deploy applications. Even more so, it is imperative to understand how to work with your own TAS environment. Every TAS installation operates the same way, but there are many ways to interact with it. Your company may have its own process for getting an account, organizations, spaces, and apps up and running.
```

Comment:
```suggestion
Even if you spend most of your time working on software code, it is essential to know how to use the TAS platform to deploy applications. Even more so, it is imperative to understand how to work with your own TAS environment. Every TAS installation operates the same way, but there are many ways to interact with it. Your company may have its own process for getting an account, organizations, spaces, and apps up and running.
```

Reply:
Let's not dictate what the reader will be doing, but instead clarify that these concepts are still important _if_ they are software developers.